src/network/netdev/netdev-gperf.gperf

```diff
@@ -129,6 +129,7 @@ Bridge.MaxAgeSec,            config_parse_sec,                     0,
 Bridge.AgeingTimeSec,        config_parse_sec,                     0,                             offsetof(Bridge, ageing_time)
 Bridge.ForwardDelaySec,      config_parse_sec,                     0,                             offsetof(Bridge, forward_delay)
 Bridge.Priority,             config_parse_uint16,                  0,                             offsetof(Bridge, priority)
+Bridge.GroupFwdMask,         config_parse_uint16,                  0,                             offsetof(Bridge, group_fwd_mask)
```

Comment:
not a fan of abbreviated settings like this. Maybe rename this "GroupForwardMask" or so?